src/SFA.DAS.Reservations.Infrastructure/HealthCheck/AccountApiHealthCheck.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.Http;
+using SFA.DAS.Reservations.Infrastructure.Extensions;
+
+namespace SFA.DAS.Reservations.Infrastructure.HealthCheck
+{
+     public class AccountApiHealthCheck : IHealthCheck
+     {
+        private const string HealthCheckResultDescription = "Account Api check";
+        private readonly IAccountApiClient _apiClient;
+        private readonly ILogger<AccountApiHealthCheck> _logger;
+
+        public AccountApiHealthCheck(
+            IAccountApiClient apiClient,
+            ILogger<AccountApiHealthCheck> logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = new CancellationToken())
+        {
+            _logger.LogInformation("Pinging Commitments API");
+
+            try
+            {
+                var timer = Stopwatch.StartNew();
+                await _apiClient.GetStatistics();
```

Comment:
Can we call GetAccount(1) instead? It doesnt matter if it returns a 404, as long as its not a 500 response code

Reply:
Updated